src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1306,7 +1283,7 @@ double UsgsAstroLsSensorModel::getImageTime(
    const csm::ImageCoord& image_pt) const
 {
    // Remove 0.5 after ISIS dependency in the linescanrate is gone

```

Comment:
Move this comment down to when the actual time is computed. That 0.5 is what needs to be removed once we change the linescan rate so that it is 0 based instead of 0.5 based.